ksa_compliance/ksa_compliance/doctype/zatca_phase_1_business_settings/zatca_phase_1_business_settings.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2024, LavaLoon and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe.model.document import Document
+
+
+class ZATCAPhase1BusinessSettings(Document):
+	# begin: auto-generated types
+	# This code is auto-generated. Do not modify anything in this block.
+
+	from typing import TYPE_CHECKING
+
+	if TYPE_CHECKING:
+		from frappe.types import DF
+
+		address: DF.Link
+		company: DF.Link
+		status: DF.Literal["Active", "Disabled"]
+		type_of_transaction: DF.Literal["Simplified", "Standard Tax Invoice", "Both"]
+		vat_registration_number: DF.Data | None
+	# end: auto-generated types
+	pass
+
+	def validate(self):
+		if frappe.get_value("ZATCA Business Settings", {"company": self.company}) and self.status == "Active":
+			frappe.throw("ZATCA Phase 2 Business Settings already enabled.", title="Another Setting Already Enabled")
```

Comment:
It's a good practice to include IDs and/or a link to the offending document in question, .e.g. `ZATCA Phase 2 Business settings already enabled for company '{company}'`.

You can generate a link using `frappe.utils.data.get_link_to_form`, e.g. `ZATCA Phase 2 Business settings already enabled for company '{company}': {link}` where `link` is `get_url_to_doc('ZATCA Business Settings', business_settings_id, business_settings_id)`